measurement/base.py:15: error: Returning Any from function declared to return "str"
measurement/base.py:89: error: Incompatible types in assignment (expression has type "None", variable has type "Union[str, Decimal]")
measurement/base.py:105: error: Unsupported operand types for * ("Decimal" and "str")
measurement/base.py:105: note: Right operand is of type "Union[str, Decimal]"
measurement/base.py:108: error: Unsupported operand types for / ("Decimal" and "str")
measurement/base.py:108: note: Right operand is of type "Union[str, Decimal]"
measurement/base.py:111: error: "None" has no attribute "replace"
measurement/base.py:181: error: Unsupported operand types for * ("str" and "Decimal")
measurement/base.py:181: note: Left operand is of type "Union[str, Decimal]"
measurement/base.py:186: error: Unsupported operand types for * ("str" and "Decimal")
measurement/base.py:186: note: Left operand is of type "Union[str, Decimal]"
measurement/base.py:191: error: Unsupported operand types for * ("str" and "Decimal")
measurement/base.py:191: note: Left operand is of type "Union[str, Decimal]"
measurement/base.py:211: error: Function is missing a type annotation
measurement/base.py:225: error: "type" has no attribute "_units"
measurement/base.py:253: error: Item "Decimal" of "Union[str, Decimal, int, None]" has no attribute "split"
measurement/base.py:253: error: Item "int" of "Union[str, Decimal, int, None]" has no attribute "split"
measurement/base.py:253: error: Item "None" of "Union[str, Decimal, int, None]" has no attribute "split"
measurement/base.py:261: error: Argument 1 to "Decimal" has incompatible type "Union[str, Decimal, int, None]"; expected "Union[Decimal, float, str, Tuple[int, Sequence[int], int]]"
measurement/base.py:264: error: Value of type "Decimal" is not indexable
measurement/base.py:265: error: "AbstractUnit" has no attribute "org_name"
measurement/base.py:270: error: Value of type "Decimal" is not indexable
measurement/base.py:276: error: Returning Any from function declared to return "Decimal"
measurement/base.py:280: error: Value of type "Decimal" is not indexable
measurement/base.py:284: error: Returning Any from function declared to return "Decimal"
measurement/base.py:296: error: Returning Any from function declared to return "Decimal"
measurement/base.py:296: error: Argument 2 to "getattr" has incompatible type "None"; expected "str"
measurement/base.py:299: error: Argument 2 to "getattr" has incompatible type "None"; expected "str"
measurement/base.py:302: error: "AbstractUnit" has no attribute "org_name"
measurement/base.py:305: error: "AbstractUnit" has no attribute "org_name"
measurement/base.py:306: error: "AbstractUnit" has no attribute "org_name"
measurement/base.py:308: error: Function is missing a type annotation
measurement/base.py:327: error: Argument 2 to "getattr" has incompatible type "None"; expected "str"
measurement/base.py:340: error: Argument 2 to "getattr" has incompatible type "None"; expected "str"
measurement/base.py:346: error: Function is missing a type annotation
measurement/base.py:348: error: "AbstractUnit" has no attribute "org_name"
measurement/base.py:349: error: "AbstractUnit" has no attribute "org_name"
measurement/base.py:355: error: Function is missing a type annotation
measurement/base.py:358: error: Function is missing a type annotation
measurement/base.py:361: error: Function is missing a type annotation
measurement/base.py:365: error: "AbstractUnit" has no attribute "org_name"
measurement/base.py:371: error: "AbstractUnit" has no attribute "org_name"
measurement/base.py:373: error: Function is missing a type annotation
measurement/base.py:376: error: Function is missing a type annotation
measurement/measures/time.py:33: error: Function is missing a type annotation
measurement/measures/time.py:36: error: Call to untyped function "__mul__" in typed context
measurement/measures/temperature.py:10: error: Only concrete class can be given where "Type[DegreeUnit]" is expected
measurement/measures/temperature.py:15: error: Function is missing a return type annotation
measurement/measures/temperature.py:16: error: "None" has no attribute "replace"
measurement/measures/temperature.py:21: error: Function is missing a type annotation
measurement/measures/temperature.py:24: error: Function is missing a type annotation
measurement/measures/temperature.py:29: error: Function is missing a type annotation
measurement/measures/temperature.py:33: error: Function is missing a type annotation
measurement/measures/geometry.py:75: error: Function is missing a type annotation
measurement/measures/geometry.py:82: error: Call to untyped function "__mul__" in typed context
measurement/measures/geometry.py:84: error: Function is missing a type annotation
measurement/measures/geometry.py:93: error: Function is missing a type annotation
measurement/measures/geometry.py:97: error: Call to untyped function "square" in typed context
measurement/measures/geometry.py:99: error: Call to untyped function "__new__" in typed context
measurement/measures/geometry.py:103: error: Function is missing a type annotation
measurement/measures/geometry.py:106: error: Incompatible types in assignment (expression has type "str", variable has type "None")
measurement/measures/geometry.py:139: error: Function is missing a type annotation
measurement/measures/geometry.py:144: error: Function is missing a type annotation
measurement/measures/geometry.py:148: error: Call to untyped function "__truediv__" in typed context
measurement/measures/geometry.py:150: error: Function is missing a type annotation
measurement/measures/geometry.py:154: error: Call to untyped function "__mul__" in typed context
measurement/measures/geometry.py:158: error: Function is missing a type annotation
measurement/measures/geometry.py:162: error: Call to untyped function "cubic" in typed context
measurement/measures/geometry.py:163: error: Call to untyped function "__new__" in typed context
measurement/measures/geometry.py:167: error: Function is missing a type annotation
measurement/measures/geometry.py:170: error: Incompatible types in assignment (expression has type "str", variable has type "None")
measurement/measures/geometry.py:246: error: Function is missing a type annotation
measurement/measures/geometry.py:251: error: Function is missing a type annotation
measurement/measures/geometry.py:258: error: Call to untyped function "__truediv__" in typed context
measurement/measures/electromagnetism.py:20: error: Function is missing a type annotation
measurement/measures/electromagnetism.py:23: error: Call to untyped function "__mul__" in typed context
measurement/measures/electromagnetism.py:33: error: Function is missing a type annotation
measurement/measures/electromagnetism.py:36: error: Call to untyped function "__mul__" in typed context
measurement/measures/electromagnetism.py:71: error: Function is missing a type annotation
measurement/measures/electromagnetism.py:78: error: Call to untyped function "__truediv__" in typed context
measurement/measures/mechanics.py:12: error: Function is missing a type annotation
measurement/measures/mechanics.py:16: error: Call to untyped function "div" in typed context
measurement/measures/mechanics.py:18: error: Call to untyped function "__new__" in typed context
measurement/measures/mechanics.py:22: error: Function is missing a type annotation
measurement/measures/mechanics.py:28: error: Incompatible types in assignment (expression has type "str", variable has type "None")
measurement/measures/mechanics.py:39: error: Function is missing a type annotation
measurement/measures/mechanics.py:51: error: Function is missing a type annotation
tests/measures/test_speed.py:119: error: "Type[Speed]" has no attribute "_units"
tests/measures/test_speed.py:122: error: "Type[Speed]" has no attribute "_units"
tests/test_utils.py:8: error: Argument 1 to "guess" has incompatible type "int"; expected "Decimal"
tests/test_utils.py:12: error: Argument 1 to "guess" has incompatible type "int"; expected "Decimal"
tests/test_utils.py:16: error: Argument 1 to "guess" has incompatible type "int"; expected "Decimal"
tests/test_utils.py:21: error: Argument 1 to "guess" has incompatible type "int"; expected "Decimal"
tests/test_base.py:32: error: Incompatible types in assignment (expression has type "str", variable has type "None")
tests/test_base.py:53: error: Incompatible types in assignment (expression has type "str", variable has type "None")
tests/test_base.py:73: error: Incompatible types in assignment (expression has type "str", variable has type "None")
tests/test_base.py:90: error: Argument "km" to "Distance" has incompatible type "float"; expected "Union[str, Decimal, int, None]"
tests/test_base.py:106: error: Argument 1 has incompatible type "**Dict[str, int]"; expected "Optional[str]"
tests/test_base.py:109: error: Call to untyped function "__eq__" in typed context
tests/test_base.py:109: error: Argument 1 has incompatible type "**Dict[str, int]"; expected "Optional[str]"
tests/test_base.py:112: error: Argument 1 has incompatible type "**Dict[str, int]"; expected "Optional[str]"
tests/test_base.py:113: error: Argument 1 has incompatible type "**Dict[str, int]"; expected "Optional[str]"
tests/test_base.py:116: error: Argument 1 has incompatible type "**Dict[str, int]"; expected "Optional[str]"
tests/test_base.py:119: error: Argument 1 has incompatible type "**Dict[str, int]"; expected "Optional[str]"
tests/test_base.py:120: error: Argument 1 has incompatible type "**Dict[str, int]"; expected "Optional[str]"
tests/test_base.py:123: error: Argument 1 has incompatible type "**Dict[str, int]"; expected "Optional[str]"
tests/test_base.py:126: error: Argument 1 has incompatible type "**Dict[str, int]"; expected "Optional[str]"
tests/test_base.py:127: error: Argument 1 has incompatible type "**Dict[str, int]"; expected "Optional[str]"
tests/test_base.py:128: error: Argument 1 has incompatible type "**Dict[str, int]"; expected "Optional[str]"
tests/test_base.py:132: error: Argument 1 has incompatible type "**Dict[str, int]"; expected "Optional[str]"
tests/test_base.py:132: error: Unsupported operand types for + ("Distance" and "str")
tests/test_base.py:136: error: Argument 1 has incompatible type "**Dict[str, int]"; expected "Optional[str]"
tests/test_base.py:137: error: Argument 1 has incompatible type "**Dict[str, int]"; expected "Optional[str]"
tests/test_base.py:138: error: Argument 1 has incompatible type "**Dict[str, int]"; expected "Optional[str]"
tests/test_base.py:141: error: Argument 1 has incompatible type "**Dict[str, int]"; expected "Optional[str]"
tests/test_base.py:142: error: Argument 1 has incompatible type "**Dict[str, int]"; expected "Optional[str]"
tests/test_base.py:143: error: Argument 1 has incompatible type "**Dict[str, int]"; expected "Optional[str]"
tests/test_base.py:147: error: Argument 1 has incompatible type "**Dict[str, int]"; expected "Optional[str]"
tests/test_base.py:147: error: Unsupported operand types for - ("Distance" and "str")
tests/test_base.py:154: error: Argument 1 has incompatible type "**Dict[str, int]"; expected "Optional[str]"
tests/test_base.py:155: error: Argument 1 has incompatible type "**Dict[str, int]"; expected "Optional[str]"
tests/test_base.py:156: error: Argument 1 has incompatible type "**Dict[str, int]"; expected "Optional[str]"
tests/test_base.py:159: error: Argument 1 has incompatible type "**Dict[str, int]"; expected "Optional[str]"
tests/test_base.py:163: error: Argument 1 has incompatible type "**Dict[str, int]"; expected "Optional[str]"
tests/test_base.py:174: error: Argument 1 has incompatible type "**Dict[str, int]"; expected "Optional[str]"
tests/test_base.py:176: error: Argument 1 has incompatible type "**Dict[str, int]"; expected "Optional[str]"
tests/test_base.py:178: error: Incompatible types in assignment (expression has type "int", variable has type "Distance")
tests/test_base.py:179: error: Argument 1 has incompatible type "**Dict[str, int]"; expected "Optional[str]"
tests/test_base.py:180: error: Argument 1 has incompatible type "**Dict[str, int]"; expected "Optional[str]"
tests/test_base.py:183: error: Argument 1 has incompatible type "**Dict[str, int]"; expected "Optional[str]"
tests/test_base.py:186: error: Argument 1 has incompatible type "**Dict[str, int]"; expected "Optional[str]"
tests/test_base.py:187: error: Argument 1 has incompatible type "**Dict[str, int]"; expected "Optional[str]"
tests/test_base.py:191: error: Argument 1 has incompatible type "**Dict[str, int]"; expected "Optional[str]"
tests/test_base.py:195: error: Argument 1 has incompatible type "**Dict[str, int]"; expected "Optional[str]"
tests/test_base.py:197: error: Argument 1 has incompatible type "**Dict[str, int]"; expected "Optional[str]"
tests/test_base.py:199: error: Incompatible types in assignment (expression has type "int", variable has type "Distance")
tests/test_base.py:200: error: Argument 1 has incompatible type "**Dict[str, int]"; expected "Optional[str]"
tests/test_base.py:201: error: Argument 1 has incompatible type "**Dict[str, int]"; expected "Optional[str]"
tests/test_base.py:203: error: Argument 1 has incompatible type "**Dict[str, int]"; expected "Optional[str]"
tests/test_base.py:204: error: Argument 1 has incompatible type "**Dict[str, int]"; expected "Optional[str]"
tests/test_base.py:208: error: Argument 1 has incompatible type "**Dict[str, int]"; expected "Optional[str]"
tests/test_base.py:211: error: Argument 1 has incompatible type "**Dict[str, int]"; expected "Optional[str]"
tests/test_base.py:212: error: Argument 1 has incompatible type "**Dict[str, int]"; expected "Optional[str]"
tests/test_base.py:213: error: Argument 1 has incompatible type "**Dict[str, int]"; expected "Optional[str]"
tests/measures/test_temperature.py:11: error: Argument "celsius" to "Temperature" has incompatible type "float"; expected "Union[str, Decimal, int, None]"
tests/measures/test_temperature.py:16: error: Argument "celsius" to "Temperature" has incompatible type "float"; expected "Union[str, Decimal, int, None]"
tests/measures/test_geometry.py:28: error: Argument "meter" to "Distance" has incompatible type "float"; expected "Union[str, Decimal, int, None]"
tests/measures/test_geometry.py:77: error: Call to untyped function "_attr_to_unit" of "Area" in typed context
tests/measures/test_geometry.py:78: error: Call to untyped function "_attr_to_unit" of "Area" in typed context
Found 144 errors in 11 files (checked 22 source files)
